packages/semaphore/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "@sripwoud/semaphore-artifacts",
+	"description": "zk-kit semaphore artifacts",
+	"license": "AGPL-3.0",
```

Comment:
Why `AGPL-3.0`. Do you have a preference in this regard?

Reply:
I find AGPL healthier for OSS.
AGPL redefines what distribution means.
For GPL it just means "providing a copy" (download, CD, USB, etc...).
For AGPL it means providing network/online access to the software.
 It is very relevant given the "cloud" era.
![image](https://github.com/privacy-scaling-explorations/snark-artifacts/assets/38692952/17f71219-9708-4386-a807-e27c08e6cabb)
Unfortunately most of the time GPL and MIT are still used by most proejcts.

Basically with AGPL anyone who accesses the software via a web service ("network distribution")built on it has the right to get the source code.
Not with GPL

Reply:
I see! Thanks